app/decorators/user_decorator.rb

```diff
@@ -36,4 +37,10 @@ def can_delete
       end
     end  
   end
+
+  def able_to_edit
```

Comment:
This looks like a name for a method returning a boolean value. Yet, you return an url.